moveit_py/src/moveit/moveit_core/robot_model/joint_model_group.h

```diff
@@ -1,7 +1,7 @@
 /*********************************************************************
  * Software License Agreement (BSD License)
  *
- *  Copyright (c) 2021, Peter Mitrano
+ *  Copyright (c) 2022, Peter David Fagan
```

Comment:
rip @PeterMitrano , there can only be one Peter :')

Reply:
:( @PeterMitrano is a good dude from what I can tell, would be honoured to have him co-author these bindings. In case there are issues with the code though it makes sense to make this update as it will be clear that I wrote the code.